pkg/sloop/processing/relationship.go

```diff
@@ -0,0 +1,126 @@
+package processing
+
+import (
+	"encoding/json"
+	"github.com/dgraph-io/badger/v2"
+	"github.com/golang/glog"
+	"github.com/golang/protobuf/ptypes"
+	"github.com/pkg/errors"
+	"github.com/salesforce/sloop/pkg/sloop/kubeextractor"
+	"github.com/salesforce/sloop/pkg/sloop/store/typed"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped"
+	"github.com/salesforce/sloop/pkg/sloop/store/untyped/badgerwrap"
+)
+
+
+
+
+//  transfer watchRec.Payload to Relationship payload: Convert watchRec to record in relationship table and save key-value pair
+func updateRelationshipTable(tables typed.Tables, txn badgerwrap.Txn, watchRec *typed.KubeWatchResult, metadata *kubeextractor.KubeMetadata) error {
```

Comment:
I'm not seeing where this is called?

Reply:
its being called here https://github.com/salesforce/sloop/pull/198/files#diff-53da0fbfd733c7b3070ab574d97678b9d282aa1fec1d64101eee49c35ea96ff3R96